<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteWidget</class>
 <widget class="QWidget" name="NoteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="frameWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Title:</string>
        </property>
        <property name="buddy">
         <cstring>editTitle</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KLineEdit" name="editTitle"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Tags:</string>
        </property>
        <property name="buddy">
         <cstring>editTags</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TagEdit" name="editTags"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Rating:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KRatingWidget" name="editRating"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Content:</string>
        </property>
        <property name="buddy">
         <cstring>editContent</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="editContent"/>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Discard|QDialogButtonBox::Save</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KRatingWidget</class>
   <extends>QFrame</extends>
   <header>kratingwidget.h</header>
  </customwidget>
  <customwidget>
   <class>TagEdit</class>
   <extends>QLineEdit</extends>
   <header>propertywidgets/tagedit.h</header>
   <slots>
    <signal>widgetEnabled(bool)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NoteWidget</receiver>
   <slot>saveNote()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NoteWidget</receiver>
   <slot>discardNote()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveNote()</slot>
  <slot>discardNote()</slot>
 </slots>
</ui>
